src/UglyToad.PdfPig/Content/Page.cs

```diff
@@ -163,6 +163,16 @@ public IReadOnlyList<Hyperlink> GetHyperlinks()
         /// </summary>
         public IReadOnlyList<MarkedContentElement> GetMarkedContents() => Content.GetMarkedContents();
 
+        /// <summary>
+        /// Convert page to image
+        /// </summary>
+        /// <param name="scale"></param>
+        /// <param name="drawingProcessor"></param>
+        public System.IO.MemoryStream ToImage(double scale, IDrawingProcessor drawingProcessor)
```

Comment:
What is the memorystream going to contain at this point, a PNG/JPG/BMP or other format, it might be best to provide a more informative return type?

Reply:
I want to give the choice of returned image format. If fine with you, I'll tackle this later on

Reply:
That's fine :+1: